ci-operator/config/openshift-assisted/assisted-installer-deployment/openshift-assisted-assisted-installer-deployment-master.yaml

```
@@ -69,7 +69,7 @@ tests:
     test:
     - ref: assisted-baremetal-tools-triage-report
 - as: snapshot-repos
-  cron: '@midnight'
+  cron: '0 * * * *'
```

linter will probably yell at you
```suggestion
  cron: 0 * * * *
```